src/platform/forms-system/test/js/review/ObjectField.unit.spec.jsx

```diff
@@ -904,4 +904,68 @@ describe('Schemaform review: ObjectField', () => {
     fireEvent.click(editButton);
     expect(onClick.called).to.be.true;
   });
+
+  it('should render custom ObjectViewField even when not attached to the root', () => {
```

Comment:
For this test we are including the parent's data in `schema`, `uiSchema`, & `idSchema` for reader context, but we are only passing the properties for the child `ObjectViewField`.